Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -305,5 +351,62 @@ public class LeapServiceProvider : LeapProvider {
 
       dest.CopyFrom(source).Transform(leapTransform);
     }
+
+    //Precull-Latching Functions
+    protected virtual void OnEnable() {
+      Camera.onPreCull -= LateUpdateHandTransforms;
+      Camera.onPreCull += LateUpdateHandTransforms;
+      _transformArray[0] = Matrix4x4.identity;
```

Comment:
This logic to reset the global shader constant can be placed in it's own method to reduce duplication.